CDTDatastore/touchdb/TD_Database+Insertion.m

```diff
@@ -560,7 +571,7 @@ - (TDStatus)forceInsert:(TD_Revision*)rev
                     return;
                 }
             } else {
-                docNumericID = [strongSelf insertDocumentID:docID inDatabase:db];
+                docNumericID = [strongSelf insertDocumentID:docID inDatabase:db error:NULL];
```

Comment:
Wouldn't we be better making use of the error to differentiate between db full errors and general db errors as done elsewhere?